api/src/main/java/org/openmrs/module/bedmanagement/service/BedManagementService.java

```diff
@@ -283,6 +283,8 @@ public interface BedManagementService extends OpenmrsService {
 	 *
 	 * @param bedType {@link BedType}
 	 * @param retireReason {@link String}
+	 * @return {@link BedType}
+	 * @since 6.2.0-SNAPSHOT
```

Comment:
Have we ever had a since version that points to a snapshot?

Reply:
updated